### DISCUSSION TITLE
Is there any documentation available for Locus-Bundle-System?

### DISCUSSION THREAD

**Discussion by dhawalbanker** [locus84/Locus-Bundle-System#14] [Q&A] [answered]
Please share the link, if any.

**Comment by locus84** [comment, marked as answer]
I've wrote sample api usage in a script.
Please refer this [link](https://github.com/locus84/Locus-Bundle-System/blob/master/Samples~/Sample/ApiSampleScript/LocusBundleSample.cs).

Let me know if you need something more

**Comment by keyserjaya** [comment]
If you guys wanted to use LocusBundleSystem using Remote Only.
1. Uncheck 'include in build' in all bundle > build Local (you only need to do this step, only when you are changing RemoteURL)
2. Check 'include in build' on any bundle you need > build remote

*P.s Works on editor, untested in build
